### DISCUSSION TITLE
How to use trained model ?

### DISCUSSION THREAD

**Discussion by Takinono** [SWivid/F5-TTS#530] [Q&A]
Hi there, this might be a stupid question but I wasn't able to find an answer.

I trained my model and I now have a reduced model.pt
What's next ? How do I use it in the infer / finetune gradio app ?

Thanks for your help !

**Comment by SWivid** [comment]
run 
`f5-tts_infer-gradio`

then click `custom_model`
filling the local path or the instructed format of hf
quite straightforward, just try it out!

**Reply by KaiKenju** [reply]
i have a question, once i have the checkpoint file after training can i inference it using python script
